pkg/system/env.go

```diff
@@ -96,6 +101,7 @@ const (
 	BuildahFormatConfigKey            = "BUILDAH_FORMAT"
 	ScpChecksumConfigKey              = "SCP_CHECKSUM"
 	RegistrySyncExperimentalConfigKey = "REGISTRY_SYNC_EXPERIMENTAL"
+	RegistrySaveExperimentalConfigKey = "REGISTRY_SAVE_EXPERIMENTAL"
```

Comment:
they are all using `sync` actually, so consider using one config key `REGISTRY_SYNC_EXPERIMENTAL`?

Reply:
But the scenarios are different. One is to synchronize the registry image in the filesystem stage, and the other is to synchronize the registry image in the build stage.